do_math_exercises/valid_creadit_card.py

```diff
@@ -0,0 +1,82 @@
+#HackerRank: https://www.hackerrank.com/challenges/validating-credit-card-number/problem?isFullScreen=true
+
+import re
+
+def check_first_number(digits):
+    if digits[0] not in ['4' ,'5', '6']:
+        return False
+    return True
+
+
+
+def check_len(digits):
+    if len(digits.replace('-', '')) != 16:
```

Comment:
What's 16 ? This is a magic number. Please comment and define by a constant variable